.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./node_modules/gts/",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "node/no-missing-require": ["off"], 
```

Comment:
What's the reason for disabling this?

Reply:
@fhinkel mind adding this to said (tracking) GitHub issue too?

Reply:
See PR description